File: database/seeders/ReportsSeeder.php
<?php

namespace Database\Seeders;

use Illuminate\Database\Console\Seeds\WithoutModelEvents;
use Illuminate\Database\Seeder;
use Illuminate\Support\Facades\DB;

class ReportsSeeder extends Seeder
{
    /**
     * Run the database seeds.
     */
    
    public function run(): void
    {
        DB::table('reports')->insert([
            [
                'photo' => 'foto1.jpg',
                'location' => 'Jakarta',
                'description' => 'Laporan kejadian di Jakarta',
                'impact' => 'Kritis',
                'vehicle_type' => 'Mobil',
                'area' => 'Komersial',
                'weather' => 'Cerah',
                'report_date' => '2023-10-01',
                'status' => 'Diterima',
            ],
            [
                'photo' => 'foto2.jpg',
                'location' => 'Bandung',
                'description' => 'Laporan kejadian di Bandung',
                'impact' => 'Tinggi',
                'vehicle_type' => 'Motor',
                'area' => 'Industri',
                'weather' => 'Hujan',
                'report_date' => '2023-10-02',
                'status' => 'Diterima',
            ],
            [
                'photo' => 'foto3.jpg',
                'location' => 'Surabaya',
                'description' => 'Laporan kejadian di Surabaya',
                'impact' => 'Sedang',
                'vehicle_type' => 'Truk',
                'area' => 'Publik',
                'weather' => 'Badai',
                'report_date' => '2023-10-03',
                'status' => 'Diproses',
            ],
            [
                'photo' => 'foto4.jpg',
                'location' => 'Batam',
                'description' => 'Laporan kejadian di Batam',
                'impact' => 'Kecil',
                'vehicle_type' => 'Bus',
                'area' => 'Perumahan',
                'weather' => 'Beraawan',
                'report_date' => '2023-10-04',
                'status' => 'Selesai',
            ],
            // Tambahkan lebih banyak data sesuai kebutuhan
        ]);
    }
}

File: database/seeders/ReportsSeeder.py
import pandas as pd
import numpy as np
from sqlalchemy import create_engine

# Buat koneksi ke database
engine = create_engine('mysql+pymysql://root:@localhost/db_lapor_park')

# Buat DataFrame untuk menyimpan data
data = pd.DataFrame({
    'photo': np.random.choice(['foto1.jpg', 'foto2.jpg', 'foto3.jpg', 'foto4.jpg'], size=100),
    'location': np.random.choice(['Jakarta', 'Bandung', 'Surabaya', 'Batam'], size=100),
    'description': np.random.choice(['Laporan kejadian di Jakarta', 'Laporan kejadian di Bandung', 'Laporan kejadian di Surabaya', 'Laporan kejadian di Batam'], size=100),
    'impact': np.random.choice(['Kritis', 'Tinggi', 'Sedang', 'Rendah'], size=100),
    'vehicle_type': np.random.choice(['Mobil', 'Motor', 'Truk', 'Bus'], size=100),
    'area': np.random.choice(['Komersial', 'Industri', 'Publik', 'Perumahan'], size=100),
    'weather': np.random.choice(['Cerah', 'Hujan', 'Badai', 'Berawan'], size=100),
    'report_date': pd.date_range(start='2023-10-01', periods=100),
    'status': ['Diterima'] * 100
})

# Simpan data ke tabel reports
data.to_sql('reports', engine, if_exists='append', index=False)

print("Data telah disimpan ke dalam tabel reports.")